com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -136,7 +158,22 @@ public Task<Void> create(IResource input) throws IOException, CompileExceptionEr
         }
         TimeProfiler.stop();
 
-        project.createTask(input, CopyCustomResourcesBuilder.class);
+        createSubTask(input, CopyCustomResourcesBuilder.class, builder);
+        index = 0;
+        for (String path : gameProjectDependencies) {
+            // initial values already have 'c' in the end
+            if (path != null && path.length() > 0) {
+                path = path.substring(0, path.length() - 1);
+                if (ROOT_NODES.length < index) {
+                    String[] tuples = ROOT_NODES[index];
+                    createSubTask(path, String.format("%s.%s", tuples[0], tuples[1]), builder);
+                }
+                else {
+                    createSubTask(path, "", builder);
```

Comment:
The tree starts here. We create tasks from ROOT_NODES then they create tasks as well... etc. etc